generator/formats.go

```diff
@@ -197,8 +196,8 @@ var customFormatters = map[string]struct{}{
 	"strfmt.UUID3":      struct{}{},
 	"strfmt.UUID4":      struct{}{},
 	"strfmt.UUID5":      struct{}{},
-	// Special
-	"io.ReadCloser": struct{}{}, // for "format": "binary"
```

Comment:
where did these go?

Reply:
customFormatters is used to determine IsCustomFormatter and decide to run a validation.
There is no validation to be carried on these interfaces.
The type mapping from "binary" to io.ReadCloser remains above and is important.

Reply:
the io.Writer is the same but for the client, so it should also remain. 
format: binary on the server is an  io.ReadCloser but on the client it's an io.Writer

Reply:
I saw Writer was for the client. Though I did not specifically test the client, I scooped through templates to figure out other usages of IsCustomFormatter than validation